src/kvirc/kernel/KviIrcConnectionServerInfo.cpp

```diff
@@ -275,6 +273,566 @@ KviBasicIrcServerInfo::~KviBasicIrcServerInfo()
 {
 }
 
+////////////////
+// User modes
+////////////////
+
+const QString & KviBasicIrcServerInfo::getUserModeDescription(QChar mode)
+{
+	switch(mode.unicode())
+	{
+		case 'O': return __tr2qs("O: Local IRC operator (LOCOP)"); break;
+		case 'a': return __tr2qs("a: Server administrator"); break;
+		case 'c': return __tr2qs("c: Recipient for cconn messages"); break;
+		case 'd': return __tr2qs("d: Recipient for server debug notices"); break;
+		case 'f': return __tr2qs("f: Recipient for full server notices"); break;
+		case 'i': return __tr2qs("i: Invisible"); break;
+		case 'k': return __tr2qs("k: Recipient for server KILL messages"); break;
+		case 'n': return __tr2qs("n: Recipient for nick changes"); break;
+		case 'o': return __tr2qs("o: IRC operator (OPER)"); break;
```

Comment:
I like that focus is brought to OPER, not entirely sure if we can get away with OP only but I like the capitalization.

I probably wouldnt use brackets anywhere instead separate with hyphen ```IRC operator - OPER``` simply because its cleaner and more readable. but not a show stopper, Ill leave that to you.